book/src/testnet-participation.md

```diff
@@ -238,6 +238,26 @@ The vote pubkey for the validator can also be found by running:
 $ solana-keygen pubkey ~/validator-vote-keypair.json
 ```
 
+#### Has my validator caught up?
+After your validator boots it make take some time to catch up with the cluster.
```

Comment:
Add comma: "After your validator boots, it make take"

Reply:
lol I did that but removed it cause I thought you'd tell me to

Reply:
I'm a big fan of commas :)
Also, just noticed make s/b may in that same sentence